api/Cargo.toml

```diff
@@ -10,6 +10,53 @@ keywords = { workspace = true }
 categories = { workspace = true }
```

Comment:
This file needs a cleanup, e.g. I don't think it should depend on `tui` and have `verbose` feature anymore